container/ubuntu16_04/layers/bazel/BUILD

```diff
@@ -26,22 +29,39 @@ language_tool_layer(
     name = "bazel-tools",
     base = "@ubuntu16_04//image",
     packages = [
+        "bash-completion",
+        "binutils",
         "patch",
         "unzip",
         "zip",
     ],
 )
 
-language_tool_layer(
-    name = "bazel-ltl",
-    additional_repos = [
-        "deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8",
-    ],
+toolchain_container(
+    name = "bazel_installer_image",
     base = "@ubuntu16_04//image",
-    keys = [
-        "@bazel_gpg//file",
+    files = [
+        "@bazel_installer//file",
     ],
     packages = [
-        "bazel",
+        "unzip",
+    ],
+)
+
+container_run_and_extract(
+    name = "bazel_tar",
+    commands = [
+        "cd /",
```

Comment:
better to put commands assuming absolute path instead of doing cd/ ?

Reply:
Done.